apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/inntektstub/InntektstubConsumer.java

```diff
@@ -89,6 +90,8 @@ public Mono<List<Void>> deleteInntekter(List<String> identer) {
     @Timed(name = "providers", tags = { "operation", "inntk_postInntekter" })
     public ResponseEntity<List<Inntektsinformasjon>> postInntekter(List<Inntektsinformasjon> inntektsinformasjon) {
 
+        log.info("Sender inntektstub: {}", Json.pretty(inntektsinformasjon));
```

Comment:
Ser du logger mye data, og inntektsinformasjon skrives også fullt ut to ganger, det er vel det som er meningen?!
Hvis du vil ha med alle felter må klassen få inn Lombok ```@ToString``` 😄 
Enda bedre, erstatte ```@Getter``` ```@Setter``` og ```@EqualsAndHashCode``` med ```@Data``` (som har alle fire).
(  ```@EqualsAndHashCode.Exclude``` beholdes dog )
Alle subklasser må også få tilsvarende endring hvis logging skal virke etter hensikten. 😺

Reply:
Ja det hender det er noe mismatch etter mappingen der som er vanskelig å feilsøke, så tenkte å vise hva som mottas fra frontend og hva som shippes videre 🤓
Åja, det fungerer greit med Json.pretty og vises korrekt i loggene nå, men kanskje cleanere å bruke ToString så man slipper å dra inn den 😄

Reply:
Du har full tydelig full kontroll med Json.pretty -- kjenner ikke den. Kjør på og glem mine kommentarer. 😄

Reply:
> Du har full tydelig full kontroll med Json.pretty -- kjenner ikke den. Kjør på og glem mine kommentarer. 😄

Hehe, men veldig greit å vite om @ToString, det har hendt jeg ikke har fått logget korrekt i noen Testnorge apper som ikke har Json dependency 😉

Reply:
Kanskje problemet er relatert til exclude som er lagt inn? Det gjøres tydeligvis noe equals operasjon som er avhengig av exclude. Men, det vil komme frem med å sammenligne utskriftene ...

Reply:
Denne gangen var det frontend som har skylda, det ble ikke sendt med det tomme tilleggsinformasjon objektet som er ønsket i inntektstub 😅 Skal være i orden nå, har sendt en del bestillinger med forskjellig tilleggsinfotype som alle gikk gjennom 👍